randovania/games/prime2/patcher/csharp_subprocess.py

```diff
@@ -80,9 +84,20 @@ def process_command(args: list[str], input_data: str, read_callback: Callable[[s
 
     work = _process_command_async(args, input_data, read_callback, additional_paths)
 
-    if IO_LOOP is None:
-        if is_windows():
-            asyncio.set_event_loop_policy(asyncio.WindowsProactorEventLoopPolicy())
-        asyncio.run(work)
-    else:
-        asyncio.run_coroutine_threadsafe(work, IO_LOOP).result()
+    try:
+        if IO_LOOP is None:
+            if is_windows():
+                asyncio.set_event_loop_policy(asyncio.WindowsProactorEventLoopPolicy())
+            asyncio.run(work)
+        else:
+            asyncio.run_coroutine_threadsafe(work, IO_LOOP).result()
+
+    except FileNotFoundError:
+        if needs_mono:
+            raise UnableToExportError(
+                "Unable to find mono.<br /><br />"
+                "Please install it from the "
+                "<a href='https://www.mono-project.com/download/stable'>official website</a>."
```

Comment:
"... and make sure it's located in PATH"? Just in case some installation decides to goof up.

Reply:
If the installation didn't add to the PATH, I'm worried it's broken more strongly.